<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataAnalyse</class>
 <widget class="QWidget" name="dataAnalyse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 255);</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>141</width>
           <height>91</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>日期</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateEdit">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>月数据汇总</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>年数据汇总</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>0</y>
           <width>111</width>
           <height>44</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>study</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>diary</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>bill</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>course</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="selectData">
            <item>
             <property name="text">
              <string>studynum</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>studytime</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>studyefficiency</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>执行结果</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QChartView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
